resources/views/admin/replies.blade.php

```diff
@@ -60,18 +60,22 @@
 
                                             <x-tables.table-data>
                                                 @if ($reply->updatedBy())
-                                                    <span class="inline-flex items-center px-2.5 py-0.5 rounded-full text-xs font-medium bg-lio-100 text-gray-800">YES</span>
+                                                    <span class="inline-flex items-center rounded-full bg-lio-100 px-2.5 py-0.5 text-xs font-medium text-gray-800"
+                                                        >YES</span
+                                                    >
                                                 @else
-                                                    <span class="inline-flex items-center px-2.5 py-0.5 rounded-full text-xs font-medium bg-gray-100 text-gray-800">NO</span>
+                                                    <span class="inline-flex items-center rounded-full bg-gray-100 px-2.5 py-0.5 text-xs font-medium text-gray-800"
+                                                        >NO</span
+                                                    >
```

Comment:
This one is a bit weird. Doesn't feels correct that these `>` chars get moved to a different line?